.travis.yml

```diff
@@ -6,14 +6,16 @@ os:
 
 before_install:
   - if [ $TRAVIS_OS_NAME == "linux" ]; then
+      export CODE_DOWNLOAD_URL="https://go.microsoft.com/fwlink/?LinkID=620884";
       export CXX="g++-4.9" CC="gcc-4.9" DISPLAY=:99.0;
       sh -e /etc/init.d/xvfb start;
       sleep 3;
+    else
+      export CODE_DOWNLOAD_URL="https://go.microsoft.com/fwlink/?LinkID=620882";
```

Comment:
Added to make sure it downloads VSCode@0.10.10

More info: https://github.com/Microsoft/vscode-extension-vscode/issues/18